packages/x-tree-view/src/TreeItem/TreeItem.types.ts

```diff
@@ -91,6 +91,7 @@ export interface TreeItemProps extends Omit<React.HTMLAttributes<HTMLLIElement>,
    * The system prop that allows defining system overrides as well as additional CSS styles.
    */
   sx?: SxProps<Theme>;
+  isBeingEdited: boolean;
```

Comment:
I would rather user the `instance` to get this info, this would allow us to now add it on the `TreeItem` props, which as explicitly used by our users on the `SimpleTreeView`.